guides/how-we-do-pitches.md

```diff
@@ -0,0 +1,80 @@
+# How we do pitches
+
+Pitching is an opportunity to demonstrate our expertise, showcase our work and demonstrate our values. Sometimes, we have plenty of notice and time to prepare, other times we have to turn pitches around quickly. To give ourselves the best possible chance, we should follow the steps outlined below.
+
+## Preparing in advance
+
+When we find out we are pitching, the first thing to do is assemble the pitch team, and make them aware as early as possible of their involvement. The people pitching should be whoever is lined up to do the work if we’re successful. We should also nominate someone to lead and coordinate producing the pitch.
+
+When we receive an invitation to a pitch, send a prompt reply back to thank the buyer for the invitation and give them any information they need (such as names of attendees).
+
+Think about the logistics. Look at all the emails or any notes from conversations with  the buyer and pull out any deadlines and important dates. Put the dates into the pitch members’ calendars and set reminders in Slack so we are all clear when the deadline is.
+
+Factor in time to rehearse the pitch, ideally a day before to give us time to make any last minute changes or more time to practice. Whoever is co-ordinating the pitch should make sure this happens.  As well as the bid team members, invite others to attend the run-through to act as an audience.
+
+Prepare some good questions for the assessment panel. There is usually an opportunity at the end of the pitch to ask some questions to the buyer.
+
+## Writing the deck
+
+Look at the original brief and pull out the evaluation criteria and weightings. These should form the initial headings for the pitch deck and the weightings should give you timings.
```

Comment:
As someone who's never been involved in pitching, I might be lacking some essential terminology: what are "weightings" in this context, and how do they relate to timings? Is it the timing of the deck cards, or the timings of the project itself?